crates/handlers/src/graphql/model/oauth.rs

```diff
@@ -21,7 +21,7 @@ use ulid::Ulid;
 use url::Url;
 
 use super::{BrowserSession, NodeType, SessionState, User, UserAgent};
-use crate::{state::ContextExt, UserId};
+use crate::graphql::{state::ContextExt, UserId};
```

Comment:
I don't have a strong opinion, but maybe we should grab all this from `super::super`? 
```suggestion
use super::super::{state::ContextExt, UserId};
```

Reply:
you're right, that's fine with me